src/index.ts

```diff
@@ -59,6 +81,31 @@ const main = async () => {
   console.error(`Initializing Dynatrace MCP Server v${VERSION}...`);
   const { oauthClientId, oauthClientSecret, dtEnvironment, dtPlatformToken, slackConnectionId } = dynatraceEnv;
 
+  // Test connection on startup
+  try {
+    console.error(`Testing connection to Dynatrace environment: ${dtEnvironment}...`);
+    await testDynatraceConnection(dtEnvironment, oauthClientId, oauthClientSecret, dtPlatformToken);
+    console.error(`Successfully connected to the Dynatrace environment at ${dtEnvironment}.`);
+  } catch (error: any) {
+    console.error(`Fatal: Could not connect to the Dynatrace environment.`);
+    if (isClientRequestError(error)) {
+      const e: ClientRequestError = error;
+      let additionalErrorInformation = '';
+      if (e.response.status == 403) {
+        additionalErrorInformation =
+          'Note: Your user or service-user is most likely lacking the necessary permissions/scopes for this API Call.';
+      }
+      console.error(
+        `Client Request Error: ${e.message} with HTTP status: ${
+          e.response.status
+        }. ${additionalErrorInformation} (body: ${JSON.stringify(e.body)})`,
+      );
+    } else {
+      console.error(`Error: ${error.message}`);
+    }
+    process.exit(1);
```

Comment:
If we do a process.exit(1) here, would that lead to the MCP server being restarted by VSCode / ...? If so, we would end up in an endless-loop that keeps contacting the Dynatrace API, but it will always respond with a `403`.